appengine/standard/firebase/firetactoe/fire_index.html

```diff
@@ -0,0 +1,304 @@
+<html>
+  <head>
+
+  <script src="https://www.gstatic.com/firebasejs/3.3.2/firebase.js"></script>
+  <!-- <script src="https://www.gstatic.com/firebasejs/3.3.0/firebase-database.js"></script> -->
+  <script>
+  	// Initialize Firebase object
+  	var config = {
+    	apiKey: "AIzaSyB_pwNiBi8Q3iakJitkQHtEQ5SZWmp9EiY",
+    	authDomain: "imposing-mind-131903.firebaseapp.com",
+    	databaseURL: "https://imposing-mind-131903.firebaseio.com"
+    	storageBucket: "imposing-mind-131903.appspot.com"
+  	};
+  	firebase.initializeApp(config);
+  </script>
+
+      <style type='text/css'>
+        body {
+          font-family: 'Helvetica';
+        }
+
+        #board {
+          width:152px; 
+          height: 152px;
+          margin: 20px auto;
+        }
+        
+        #display-area {
+          text-align: center;
+        }
+        
+        #this-game {
+          font-size: 9pt;
+        }
+        
+        #winner {
+        }
+        
+        table {
+          border-collapse: collapse;
+        }
+        
+        td {
+          width: 50px;
+          height: 50px;
+          font-family: "Helvetica";
+          font-size: 16pt;
+          text-align: center;
+          vertical-align: middle;
+          margin:0px;
+          padding: 0px;
+        }
+        
+        div.cell {
+          float: left;
+          width: 50px;
+          height: 50px;
+          border: none;
+          margin: 0px;
+          padding: 0px;
+        }
+        
+        div.mark {
+          position: absolute;
+          top: 15px;          
+        }
+        
+        div.l {
+          border-right: 1pt solid black;
+        }
+        
+        div.c {
+        }
+        
+        div.r {
+          border-left: 1pt solid black;
+        }
+        
+        div.t {
+          border-bottom: 1pt solid black;
+        }
+        
+        div.m {
+        }
+        
+        div.b {
+          border-top: 1pt solid black;
+        }
+      </style>
+  </head>
+  <body>
+    <script type='text/javascript'>
+      var state = {
+        game_key: '{{ game_key }}',
+        me: '{{ me }}'
+      };
+
+      // This is our Firebase realtime DB path that we'll listen to for updates
+      // We'll initialize this later in openChannel()
+      var channel = null;
+
+      updateGame = function() {
+        for (i = 0; i < 9; i++) {
+          var square = document.getElementById(i);
```

Comment:
I feel that jquery would make this section slightly more readable, please feel free to use it if you feel the same.